Server/copy-assets.ts

```diff
@@ -0,0 +1,37 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import * as util from 'util';
+
+const mkdir = util.promisify(fs.mkdir);
```

Comment:
I think you could use fs/promises here instead of `promisify` everything by hand. 
https://nodejs.org/api/fs.html#promises-api